drivers/gles3/storage/light_storage.h

```diff
@@ -33,6 +33,10 @@
 
 #ifdef GLES3_ENABLED
 
+// This must come first to avoid windows.h mess
+#include "platform_config.h"
```

Comment:
I think all those `platform_config.h` includes can likely be removed. They were there for `OPENGL_INCLUDE_H`, which you removed.

And then the `windows.h` concerns are likely solved automatically.